### PR TITLE
adds license guidelines, fixes #2054

### LINKED ISSUE #2054: License policy?
After a brief chat in gitter, it seems like it would be worthwhile to have a conversation here about whether or not to have some guidelines about tool licenses for things wrapped by IUC.

I could not find an explicit policy, but maybe there should be one - or an explicit statement that there is no policy.

Should IUC tools be licensed with an [OSI approved](https://opensource.org/licenses) license?

### PR DESCRIPTION
**Pull request by bwlang**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [x] - This PR does something else (explain below)

FOR REVIEWER:
* [ ] .shed.yml file ok
    - [ ] Toolshed user `iuc` has access to associated toolshed repo(s)
* [ ] Indentation is correct (4 spaces)
* [ ] Tool version/build ok
* [ ] `<command/>`
  - [ ] Text parameters, input and output files `'single quoted'`
  - [ ] Use of `<![CDATA[ ... ]]>` tags
  - [ ] Parameters of type `text` or having `optional="true"` attribute are checked with `if str($param)` before being used
* [ ] Data parameters have a `format` attribute containing datatypes recognised by Galaxy
* [ ] Tests
  - [ ] Parameters are reasonably covered
  - [ ] Test files are appropriate
* [ ] Help
  - [ ] Valid restructuredText and uses `<![CDATA[ ... ]]>` tags
* [ ] Complies with other best practice in [Best Practices Doc](http://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html)

I think tools-iuc have this policy already, this just makes it more explicit than just the PR checklist.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -66,4 +66,5 @@ assistance).
 * The tests must pass (`planemo test --install_galaxy my_tool.xml`)
 * The tools should pass linting by planemo (`planemo lint my_tool.xml`)
 * If there's a relevant paper for the tool, it should be cited in a [citation](https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-citations) block
+* The tool must be licensed to allow use by anyone. The OSI maintains a [list of appropriate](https://opensource.org/licenses/alphabetical).
```
Comment (nsoranzo) [inline CONTRIBUTING.md]: Add `licenses` after `appropriate`?

**Comment by bgruening** [comment]
@galaxyproject/iuc is that ok? Please comment.

**Comment by peterjc** [comment]
Do we need a minimum quorum from the IUC, or shall we go ahead and merge this policy change?

**Comment by peterjc** [comment]
I think we likely have a majority approval (I'm not sure how many of the current IUC membership are active), and no objections, so I'm going to go ahead and merge this now.

**Comment by martenson** [comment]
@peterjc afaik the active list is this: https://galaxyproject.org/iuc/

**Comment by peterjc** [comment]
Thanks, that lists 18 active members, and we had almost half approve here - not quite a majority after all, but a far higher threshold than we normally use for merging pull requests (but I recognise this one was not a simple technical choice).

**Comment by martenson** [comment]
@nsoranzo is on board I think, so you have 10

**Comment by nsoranzo** [comment]
Yep, no worries!

**Pull request merged**